AppImage Automatic Updater (#457)
owncloud/docs-client-desktop: modules/ROOT/pages/automatic_updater.adoc

=== Linux

* Linux distributions provide their own update tools. +

> The ownCloud Desktop App use the Linux operating system do not perform any updates on their own. The client will inform you (menu:Settings[General > Updates]) when an update is available.

Reviewer: ```suggestion
The ownCloud Desktop app on a Linux operating system does not perform any updates on its own. The client will inform you (menu:Settings[General > Updates]) when an update is available.
```

* AppImages +

> AppImages provide an auto update feature. The check process takes place when (re)starting the AppImage. If a new AppImage is found, you get the following example screen where you can decide to install the new image or not. Updates are in-place and do not produce a new filename.

Reviewer: ```suggestion
AppImages provide an auto-update feature. The check process takes place when (re)starting the AppImage. If a new AppImage is found, you get the following example screen where you can decide whether to install the new image or not. Updates are in-place and do not produce a new file name.
```
